build/shared/lib/languages/PDE.properties

```diff
@@ -445,6 +445,18 @@ ensure_exist.messages.unrecoverable.description = Could not properly re-save the
 # Check name
 check_name.messages.is_name_modified = The sketch name had to be modified. Sketch names can only consist\nof ASCII characters and numbers (but cannot start with a number).\nThey should also be less than 64 characters long.
 
+# External changes detector
+change_detect.reload.title=Tab modified externally
+change_detect.reload.question="%s" was modified by another program.
+change_detect.reload.comment=Would you like to keep this version or load the new changes?\nEither way, the version you discard will be saved to your sketch folder.
```

Comment:
@benfry I know you proposed this message before, but I think it's confusing. 

![image](https://cloud.githubusercontent.com/assets/3177098/25726696/adfed958-3125-11e7-8d58-8b9d3aa740be.png)

Something along these lines seems more clear:

![image](https://cloud.githubusercontent.com/assets/3177098/25726716/c89c8986-3125-11e7-981d-360180b861f4.png)